windows-iotcore/develop-your-app/BuildingAppsForIoTCore.md

```diff
@@ -105,8 +105,6 @@ Traditional UWP languages ship with support in Visual Studio by default. All of
  The IoT targeted languages require the download of the "Windows IoT Core Project Templates" from the Visual Studio **Tools->Extensions and Updates** manager.  The IoT Focused languages support only Background Applications. You can also build *Windows Runtime Components* using C#, C++, or Visual Basic and then reference those libraries from any other language (except Python).
 * Languages
  * Arduino Wiring
```

Comment:
just one "other" languages?

Reply:
Should we scrap it altogether? Or do you mean just to rephrase?

Reply:
Possibly rephrase.  I'm seeing it out of context, but it seems weird to have a list with one thing in it